### PR TITLE
[UI] Use tabbed pane with Queries and Executors tabs

### PR DESCRIPTION
**Pull request by andygrove**
# Which issue does this PR close?

N/A

 # Rationale for this change

The list of executors could be very large so it does not make sense to show in the header. 

# What changes are included in this PR?

- Use tabbed interface with Jobs tab and Executors tab.

## Jobs Tab

tab0

## Executors Tab

tab

# Are there any user-facing changes?

### CODE REVIEW

**Comment by andygrove** [comment]
@KenSuenobu @onthebridgetonowhere Could you review this PR when you get a chance?

File: ballista/ui/scheduler/src/App.tsx [hunk 1]
```diff
@@ -50,20 +60,43 @@ const App: React.FunctionComponent<any> = () => {
...
-  console.log(JSON.stringify(schedulerState));
-
   return (
     <Box>
       <Grid minH="100vh">
         <VStack alignItems={"flex-start"} spacing={0} width={"100%"}>
           <Header schedulerState={schedulerState} />
           <Summary schedulerState={schedulerState} />
-          <QueriesList queries={jobs} />
+          <Tabs>
```
Comment (onthebridgetonowhere) [inline ballista/ui/scheduler/src/App.tsx:86]: We probably want to have here: `<Tabs width={"100%"}>`

File: ballista/ui/scheduler/src/components/ExecutorsList.tsx [hunk 2]
```diff
@@ -56,16 +61,12 @@ const columns: Column<any>[] = [
...
-
-export const NodesList: React.FunctionComponent<NodesListProps> = ({
-  nodes = [],
+export const ExecutorsList: React.FunctionComponent<ExecutorsListProps> = ({
+  executors = [],
 }) => {
   return (
     <Box flex={1}>
-      <DataTable maxW={960} columns={columns} data={nodes} pageSize={4} />
+      <DataTable maxW={960} columns={columns} data={executors} pageSize={4} />
```
Comment (onthebridgetonowhere) [inline ballista/ui/scheduler/src/components/ExecutorsList.tsx:69]: The `maxW` property is not really useful, unless we need this width size restriction. Removing it makes the table look better.

**Review by onthebridgetonowhere — CHANGES_REQUESTED** [review]

**Comment by onthebridgetonowhere** [comment]
> @KenSuenobu @onthebridgetonowhere Could you review this PR when you get a chance?

@andygrove it looks great. I have only two comments regarding look&feel, see above.

**Comment by andygrove** [comment]
Thanks for the review @onthebridgetonowhere. I addressed those two points and it looks much better now.

**Review by onthebridgetonowhere — APPROVED** [review]

**Pull request merged**